manifests/application.pp

```diff
@@ -66,6 +66,8 @@
   $postgresql_evr_package = $katello::params::postgresql_evr_package
   $manage_db = $foreman::db_manage
 
+  $pulp2_support = $facts['os']['release']['major'] == '7'
```

Comment:
Should this be a parameter? That way in `init.pp` we can also set it. This may be useful in the future to test on pure Pulp 3 setups on EL7.

Reply:
Like this?